packages/renderer/src/lib/network/NetworkDetails.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+import { StatusIcon, Tab } from '@podman-desktop/ui-svelte';
+import { router } from 'tinro';
+
+import { networksListInfo } from '/@/stores/networks';
+
+import Route from '../../Route.svelte';
+import VolumeIcon from '../images/VolumeIcon.svelte';
+import DetailsPage from '../ui/DetailsPage.svelte';
+import { getTabUrl, isTabSelected } from '../ui/Util';
+import { NetworkUtils } from './network-utils';
+import NetworkActions from './NetworkActions.svelte';
+import NetworkDetailsInspect from './NetworkDetailsInspect.svelte';
+import NetworkDetailsSummary from './NetworkDetailsSummary.svelte';
+import type { NetworkInfoUI } from './NetworkInfoUI';
+
+interface Props {
+  networkName: string;
+  engineId: string;
+}
+
+let { networkName, engineId }: Props = $props();
+
+let network: NetworkInfoUI | undefined = $state();
+let detailsPage: DetailsPage | undefined = $state();
+
+const networkUtils = new NetworkUtils();
+
+let matchingNetwork = $derived(
+  $networksListInfo.find(network => network.Name === networkName && network.engineId === engineId),
+);
+
+$effect(() => {
+  if (matchingNetwork) {
+    network = networkUtils.toNetworkInfoUI(matchingNetwork);
+  } else if (detailsPage) {
+    detailsPage.close();
+  }
+});
+</script>
+
+{#if network}
+  <DetailsPage title={network.name} subtitle={network.shortId} bind:this={detailsPage}>
+    {#snippet iconSnippet()}
+      <StatusIcon icon={VolumeIcon} size={24} status={network?.status} />
+    {/snippet}
+    {#snippet actionsSnippet()}
+      {#if network}
+        <NetworkActions object={network} detailed={true} />
+      {/if}
+    {/snippet}
+    {#snippet tabsSnippet()}
+      <Tab title="Summary" selected={isTabSelected($router.path, 'summary')} url={getTabUrl($router.path, 'summary')} />
+      <Tab title="Inspect" selected={isTabSelected($router.path, 'inspect')} url={getTabUrl($router.path, 'inspect')} />
+    {/snippet}
+    {#snippet contentSnippet()}
+      {#if network}
```

Comment:
suggestion(non-blocking): I think we can remove this if block, seems that typecheck is passing and network details tabs displayed ok on my Mac

```suggestion
     {#snippet contentSnippet()}
      <Route path="/summary" breadcrumb="Summary" navigationHint="tab">
        <NetworkDetailsSummary network={network} />
      </Route>
      <Route path="/inspect" breadcrumb="Inspect" navigationHint="tab">
        <NetworkDetailsInspect network={network} />
      </Route>
    {/snippet}
```